domains/auth/Auth.API/API/Controllers/LoginController.cs

```diff
@@ -28,4 +51,92 @@ public LoginController(IOidcService oidcService)
 
         return oidcService.CreateAuthorizationUri(state);
     }
+
+    [HttpGet]
+    [Route("/oidc/login/callback")]
```

Comment:
Shouldnt it be `auth/oidc/login/callback` according to this `https://github.com/Energinet-DataHub/energy-origin/blob/main/doc/api/auth.md#oidc-login-callback`?